test/build_model.py

```diff
@@ -588,6 +588,26 @@ def test_013_set_onnx_opset(self):
         model_opset = getattr(onnx_model.opset_import[0], "version", None)
         assert user_opset == model_opset
 
+    def test_014_export_only(self):
+        build_name = "export_only"
+
+        user_opset = 15
+        assert user_opset != build.DEFAULT_ONNX_OPSET
```

Comment:
Why are we comparing against opset15?

Reply:
Because I copy pasted the opset test to make this test and forgot to delete those two lines 👀 

fixed in https://github.com/groq/mlagility/pull/306/commits/5eb18eb545038147c9d2b857a0db727b73ee76b6